src/components/style.css

```diff
@@ -0,0 +1,213 @@
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+}
+
+body {
+  background: #3295a7;
+  font-family: sans-serif;
+}
+
+header {
+  width: 100%;
+  height: 390px;
+}
+
+.map-image {
+  position: absolute;
+  width: 70%;
+  left: 20px;
+}
+
+.info-statusTxt {
+  margin-left: 10px;
+}
+
+span {
+  position: absolute;
+  left: 10px;
+  color: #000;
+}
+
+/* Home */
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #215f81;
+  height: 40px;
+  padding-top: 7px;
+}
+
+.left-display {
+  display: flex;
+  justify-content: space-between;
+  width: 35%;
+}
+
+.topic {
+  color: white;
+  margin-left: 7%;
+}
+
+.right-display {
+  display: flex;
+  margin-right: 50px;
+  gap: 33%;
+}
+
+.home,
+.microphone,
+.setting {
+  width: 30px;
+  height: 30px;
+}
+
+.home {
+  padding-left: 20px;
+}
+
+.background-image {
+  height: 310px;
+  z-index: -1;
+  width: 100%;
+}
+
+.input-area {
+  text-align: center;
+  z-index: 10;
+  margin-top: -195px;
+}
+
+.search-bar {
+  padding: 0.8%;
+  width: 39%;
+  font-size: 16px;
+  border: none;
+}
+
+.topic-2 {
+  width: 100%;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  background: #2f6a8a;
+  margin-top: -33px;
+}
+
+.topic-text {
+  font-size: 20px;
+  padding-left: 10px;
+  color: white;
+}
+
+.countries {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  gap: 1px;
+  margin-top: 1px;
+}
+
+.country-list {
+  background: #1b506d;
+  width: 100%;
+  position: relative;
+  height: 210px;
+}
```

Comment:
Great design @kamranahmad123 🤝

- [ ] As stated in the project requirements, you are supposed to follow the design guidelines provided. Kindly ensure your design replicates the given design as much as possible. Your project has four columns instead of two which is provided in the design guidelines.
- Your design
![your-design](https://github.com/kamranahmad123/Country-Details/assets/63917725/aa38fea1-17dd-482a-9ae2-ba3f975034ff)
- Given design
![given-design](https://github.com/kamranahmad123/Country-Details/assets/63917725/a291235e-eebc-4e20-950d-8458846dd05c)

Reply:
- [ ] Well resolved 🥇 
<img width="320" alt="Screenshot 2023-05-13 at 15 03 05" src="https://github.com/kamranahmad123/Country-Details/assets/58692590/9d2136c3-fa8b-4f6d-992c-572feb310968">